Makefile

```diff
@@ -36,6 +36,14 @@ bench:
 	$(info INFO: Starting build $@)
 	go test -bench=.
 
+alpha-test-%: release/goss-%
```

Comment:
When does this get executed?

Reply:
via `ci/build.sh` in the if darwin or windows bit.

(Honestly, Makefiles are not a strong suit for me, so I haven't wired there)